cloud-controller-manager/do/resources.go

```diff
@@ -136,7 +128,33 @@ func (r *ResourcesController) sync() error {
 		if err != nil {
 			return fmt.Errorf("failed to create tag %q: %s", r.clusterIDTag, err)
 		}
+
+		// Try tagging again, which should not fail anymore due to a missing
+		// tag.
+		err = r.tagResources(res)
+	}
+
+	if err != nil {
+		return fmt.Errorf("failed to tag LB resource(s) %v with tag %q: %s", res, r.clusterIDTag, err)
```

Comment:
Won't this cause the error to be `failed to tag LB resource(s) %v with tag %q: %s: failed to tag LB resource(s) %v with tag %q: %s` if tagResources fails?

Reply:
Oh yes it would. I pushed another commit to address that.